crates/bevy_ui/src/ui_node.rs

```diff
@@ -20,6 +20,16 @@ pub struct Node {
 }
 
 /// An enum that describes possible types of value in flexbox layout options
+///
+/// There are numeric traits implemented for this enum so you can multiply, divide, add, and subtract
```

Comment:
These changes are unrelated; please remove them :)

Reply:
oh sorry! that must have slipped in somehow!

Reply:
Due to a bug in GitHub I had to create another PR for this: #5628